src/lib/const.ts

```diff
@@ -376,11 +381,36 @@ p {
 	margin: 1em 0;
 }`.trim();
 
+export const defaultHtmlCode = `
+<!--
+Type or paste some HTML here to learn more about
+the static analysis that ESLint can do for you.
+
+The tabs are:
+
+- AST - The Abstract Syntax Tree of the code, which can
+be useful to understand the structure of the code. You
+can view this structure as JSON or in a tree format.
+-->
+
+<!DOCTYPE html>
+<html lang="ko">
```

Comment:
Most users will be in the US. :smile:
```suggestion
<html lang="en">
```